argh/src/lib.rs

```diff
@@ -817,6 +820,11 @@ pub fn parse_struct_args(
             continue;
         }
 
+        if (next_arg == "--complete" || next_arg == "complete") && !options_ended {
```

Comment:
And good point about the backwards incompatibility. I think this is going to need iterating.

As mentioned in the other comment, this would require going back and adding this switch to every command/subcommand in the ffx tool (which is what I'm targeting to test this stuff) ?